services/entity-server/src/root.js

```diff
@@ -6,26 +6,27 @@ import {
   Redirect
 } from "react-router-dom";
 import ErrorBoundary from "./components/common/error-boundary";
-import SessionView from "./views/session/session-view";
+import SessionView from "./views/entities/session/session-view";
 import SearchView from "./views/search/search-view";
-import { Header } from "./components/common/header";
-import { Footer } from "./components/common/footer";
-import { HomeView } from "./views/home/home-view";
-import { DataView } from "./views/data/data-view";
+import {Header} from "./components/common/header";
+import {Footer} from "./components/common/footer";
+import {HomeView} from "./views/home/home-view";
+import {DataView} from "./views/data/data-view";
```

Comment:
is this your auto-formatter and mine having a war ? :)

Reply:
I adapted my auto-formatter; Let's see where else this surfaces :)